otter/convergence/selfheal.py

```diff
@@ -0,0 +1,169 @@
+"""
+Self heal service
```

Comment:
Does the description in the service object: `A service that triggers convergence on all the groups on interval basis. Only one node is allowed to do this.` serve the purpose?